### PR TITLE
Improve warnings in case of incompatibility

### PR DESCRIPTION
**Pull request by EvgeniyaKatunina**
Make them shorter and simpler.

### CODE REVIEW

File: org.eclipse.cdt.cross.arc.gnu/src/org/eclipse/cdt/cross/arc/gnu/common/StateChecker.java [hunk 1]
```diff
@@ -88,13 +88,10 @@ void checkPluginVersions(final IProject project, boolean isNew) {
...
-        ? String.format("Compatibility issues are possible. "
-            + "Your compiler plug-in' state is %s, but this project was created with an "
-            + "older compiler plug-in' state.", currentState)
-        : String.format("Compatibility issues are possible. "
-            + "Your compiler plug-in' state is %s, but this project was created with a "
-            + "compiler state %s.", currentState, projectState);
+    final String warningMsg = String.format(
+        "This project was created with an incompatible version of ARC plug-in."
+        + " Supported version of a project file is %s, but it is %s.",
+        currentState, projectState.equals(UNREAL_STATE) ? "an older one" : projectState);
```
Comment (anthony-kolesov) [inline org.eclipse.cdt.cross.arc.gnu/src/org/eclipse/cdt/cross/arc/gnu/common/StateChecker.java]: Please surround ?: expression with ( ).

**Comment by EvgeniyaKatunina** [comment]
What about just moving argument to a new line, I think it looks better (pushed this)?

**Comment by anthony-kolesov** [comment]
It is better than what was there before, but braces have better visibility and will not be removed by automated code formatter or something like that.

**Comment by EvgeniyaKatunina** [comment]
Ok, done.

**Comment by anthony-kolesov** [comment]
Ok, please merge.

**Pull request merged**